nixos/modules/services/networking/smokeping.nix

```diff
@@ -73,19 +73,19 @@ in
         description = "Configuration for alerts.";
       };
       cgiUrl = mkOption {
-        type = types.string;
+        type = types.str;
         default = "http://${cfg.hostName}:${builtins.toString cfg.port}/smokeping.cgi";
```

Comment:
Hmm, this line seems problematic. It will cause documentation rebuilds if these values are in ones configuration.

CC @fadenb